design/metadata-handling.md

```diff
@@ -597,15 +608,21 @@ metadata:
     name: nodepool-1
 spec:
   index: 0
+  templateReference: SharedRef
```

Comment:
Is there a need to add that to the metal3dataclaim too ? My guess would be no, considering that the claims are not shared between M3DTs as the Metal3Data are, right ?

Reply:
Yes, no need to add it to dataclaim as you correctly pointed out.